src/components/form/.storybook/decorators.tsx

```diff
@@ -15,7 +15,7 @@ export const FORM_TEST_DECORATOR: Decorator = (Story) => {
     >((acc, [key, value]) => ({ ...acc, [key]: value }), {});
   };
   return (
-    <form onChange={() => setCount(count + 1)} aria-label="form">
+    <form onChange={() => setCount(count + 1)} aria-label="form" style={{width: '100%'}}>
```

Comment:
Not sure if 'form' is enough of a meaningful label for aria-label here, and if it is actually even necessary, since this is already a semantic element with meaningful inputs inside.